scripts/createComponent.js

```diff
@@ -0,0 +1,156 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+const fs = require('fs');
+const copyrightBannerScss = require('./copyrightLinter').copyrightBannerScss;
```

Comment:
You can import it like this:
`const { copyrightBannerScss, copyrightBannerHtml, copyrightBannerJs } = require('./copyrightLinter')`

Reply:
Done